stock_move_location/wizard/stock_move_location_line.py

```diff
@@ -53,17 +53,20 @@ class StockMoveLocationWizardLine(models.TransientModel):
     )
 
     @api.model
-    def get_rounding(self):
+    def _get_rounding(self):
```

Comment:
@sebalix Shouldn't rounding come from product_uom_id field ?

Reply:
@rousseldenis indeed it should, I'm looking how to convert the rounding precision to decimal precision, if there is a method in Odoo for that.

Reply:
ok, I need to use the `precision_rounding` parameter of `float_compare`, as simple as that.